### PR TITLE
allow customize label by slot

### PR DESCRIPTION
**Pull request by vanhoavn**
This allow us to customize labels by using slot:

```
                <v-checkbox>
                    <span slot="label">this is <b>important</b> field</span>
                </v-checkbox>
```

### CODE REVIEW

File: src/mixins/input.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default {
 
       if (this.id) data.attrs = { for: this.id }
 
-      return this.$createElement('label', data, this.label)
+      return this.$createElement('label', data, this.$slots.label || this.label)
```
Comment (johnleider) [inline src/mixins/input.js]: This should pass label through as a prop.

Comment (vanhoavn) [inline src/mixins/input.js]: You mean functional slots?

Comment (vanhoavn) [inline src/mixins/input.js]: `this.label` is the props provided by user, which is used for simple string input, while label slot `this.$slots.label` is used for rendering more complex labels.

Comment (johnleider) [inline src/mixins/input.js]: Valid point, I just realized my mistake.

**Pull request merged**